indico/modules/events/timetable/legacy.py

```diff
@@ -41,7 +41,7 @@ def serialize_timetable(self, event, days=None, hide_weekends=False):
             timetable[date_str] = {}
         query_options = (defaultload('contribution').subqueryload('person_links'),
                          defaultload('session_block').subqueryload('person_links'))
-        query = (event.timetable_entries
+        query = (TimetableEntry.query.with_parent(event)
```

Comment:
Because of the additional ordering on the relationship entries were not ordered as you would expect.
```
event_new = db.relationship(
        'Event',
        lazy=True,
        backref=db.backref(
            'timetable_entries',
            order_by=lambda: TimetableEntry.start_dt,
            cascade='all, delete-orphan',
            lazy='dynamic'
        )
    )
```